query/src/main/scala/filodb/query/exec/PeriodicSamplesMapper.scala

```diff
@@ -378,7 +384,8 @@ class SlidingWindowIterator(raw: RangeVectorCursor,
     */
   private def shouldAddCurToWindow(curWindowStart: Long, cur: TransientRow): Boolean = {
     // cur is inside current window
-    cur.timestamp >= curWindowStart
+    val windowStart = if (FiloQueryConfig.isInclusiveRange) curWindowStart else curWindowStart + 1
+    cur.timestamp >= windowStart
```

Comment:
Shouldn't we only need to change the initial start timestamp? Then there's no need to update this (or any other) logic?

The assumption I'm making is that this is always true:
```
curWindowEnd - curWindowStart == adjustedWindowDuration
```

Reply:
fix is to revert the new start-inclusive behavior if config is set to false